### PR TITLE
feat:Add live chat.

### PR DESCRIPTION
**Pull request by hilaldedek**

### CODE REVIEW

File: client/blog/src/pages/ChatPage.vue [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+  },
+  mounted() {
+    this.getMessages();
+  },
+  methods: {
+    async getMessages() {
+      const getToken = localStorage.getItem("access_token");
+      try {
+        const response = await fetch(
+          `http://127.0.0.1:5000/chat/${this.username}`,
```
Comment (DurmazDev) [inline client/blog/src/pages/ChatPage.vue:54]: I think domain names should not be hard-coded & some print's, console.log's forgotten.

File: api/app.py [hunk 2]
```diff
@@ -47,6 +52,86 @@
...
+            if username not in room_users.get(room, []):
+                data="Yeni bir mesajınız var"
+                emit("notification",data , to=username)
+        pass
+        message = Message.objects(roomName=room).first()
+        if message:
+            updated_messages = message.messages + dbData
+            message.update(messages=updated_messages)
+        else:
+            new_message = Message(
```
Comment (DurmazDev) [inline api/app.py:130]: Saving messages and message rooms in a database that can be created by anyone as many times as desired can be costly and may cause the database to grow. Maybe a cleaning mechanism could be written.

**Comment by dilekyalcin** [comment]
I agree with what Ahmet said, otherwise LGTM

File: api/config.py [hunk 3]
```diff
@@ -17,6 +17,7 @@ class BaseConfig(object):
     JWT_SECRET_KEY = "!6uad-[aadşawlmdşwaödDArKZ8zp0V3bfXzWxW:ffhn7JlszRG9e;d6-HcCya"
```
Comment (gecicidegisken) [inline api/config.py:17]: JWT_SECRET_KEY should not be exposed to public. It's a crucial security issue.

File: api/tests/confest.py [hunk 4]
```diff
@@ -109,6 +109,36 @@ def new_user_create():
...
+            username=second_user_data.get("username"),
+            email=second_user_data.get("email"),
+            password=second_user_data.get("password"),
+        )
+        new_user.set_password(password=second_user_data.get("password"))
+        new_user.save()
+        return new_user
+
+
+def new_user_create_3():
```
Comment (gecicidegisken) [inline api/tests/confest.py:128]: I don't get why 3 versions of the same function is needed. Could use a different approach here

**Review by gecicidegisken — CHANGES_REQUESTED** [review]

**Pull request closed without merging**